modular_skyrat/modules/blueshield/code/modsuit/mod_type.dm

```diff
@@ -3,15 +3,15 @@
 	icon = 'modular_skyrat/modules/blueshield/icons/praetorian.dmi'
 	icon_state = "praetorian-control"
 	theme = /datum/mod_theme/blueshield
-	applied_cell = /obj/item/stock_parts/cell/super
+	applied_cell = /obj/item/stock_parts/cell/high // Bubberstation Edit - Original: Super Cell
 	applied_modules = list(
-		/obj/item/mod/module/storage/large_capacity,
+		/obj/item/mod/module/storage, // Bubberstation Edit - Original: Extended storage
 		/obj/item/mod/module/magnetic_harness,
 		/obj/item/mod/module/flashlight,
-		/obj/item/mod/module/jetpack,
-		/obj/item/mod/module/projectile_dampener,
+		// /obj/item/mod/module/jetpack, // Bubberstation Removal
```

Comment:
If the statement is that you wanted blueshields to be good at rescuing in space, they should have this module.

Reply:
I'll lower the EVA armor then